simulator/src/data/save.js

```diff
@@ -377,6 +377,10 @@ export default async function save() {
 
         $('body').append(form);
         form.submit();
+        const url_projectName = projectName.replace(/\s+/g, '-').toLowerCase();
```

Comment:
This won't always be the project slug, say if two projects have the same name, then it will be suffixed by some random string.